src/api/Main/DataLayer/Shrooms.DataLayer/Migrations/201909261209583_SeedWelcomeKudos.cs

```diff
@@ -0,0 +1,21 @@
+namespace Shrooms.DataLayer.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+
+    public partial class SeedWelcomeKudos : DbMigration
+    {
+        public override void Up()
+        {
+            Sql(
+                "INSERT INTO KudosTypes" +
+                "(Name, Value, Created, Modified, Description, Type, IsActive)" +
+                "VALUES ('Welcome', 1, GETDATE(), GETDATE(), 'kudos.typeWelcome', 5, 0)"
```

Comment:
Can you change it according to currently used values in production:
Value = 3
Comment = 'Welcome Kudos for you. Be active and earn more!'
IsActive = ..couldbe be left as 0 by default, but we need to remember to update it in production, because it should be working there